app/services/doc_auth/acuant/request.rb

```diff
@@ -116,6 +116,15 @@ def create_http_exception(http_response)
         DocAuth::RequestError.new(message, http_response.status)
       end
 
+      def create_error_response(error, exception)
+        DocAuth::Response.new(
```

Comment:
Couple thoughts:

- Nit: Should `error` be `errors` to maintain semantics that it's an object of potentially-many errors?
- Would it be better to make this a single hash parameter to make call sites clearer what is being passed? i.e. `(errors:, exception:)`
- Should this be the responsibility of a subclass of `DocAuth::Response`, similar to what's done with `DocAuth::Response::TrueIdResponse`?

Reply:
1. Yeah, fixed that inconsistency.
2. I could see it but the calls are right next to each other and this method isn't going to be a widely used method so it seems a little overly verbose.
3. I thought a lot about how to handle this more simply through inheritance but it kept turning into a larger refactor than I wanted for this relatively small thing this close to release.